reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -1816,8 +1819,12 @@ public T block(Duration timeout) {
 	 * <p>
 	 * <img class="marble" src="doc-files/marbles/cacheWithTtlForMono.svg" alt="">
 	 * <p>
-	 * First subscription after the cache has been emptied triggers cache loading (ie
-	 * subscription to the source), which cannot be cancelled.
+	 * Cache loading (ie. subscription to the source) can happen either at the very
+	 * first subscription or after the cache has been cleared due to expiry.
+	 * Once that upstream subscription is started, it cannot be cancelled.
+	 * The operator will however prevent multiple concurrent subscriptions from triggering
+	 * duplicated loading (only one load-triggering subscription can win at a time,
+	 * and the others will get notified of the newly cached value when it arrives).
```

Comment:
How about the following:

```suggestion
	 * Cache loading (ie. subscription to the source) is triggered by the very 
	 * first subscription to empty or expired cache.
	 * Once that upstream subscription is started, it cannot be cancelled.
	 * The operator guarantees that only a single cash loading may happen at a time 
	 * (only one load-triggering subscription can win at a time,
	 * and the others will get notified of the newly cached value when it arrives).
```

Reply:
same should be applied to the following similar changes if makes sence

Reply:
good suggestion. I actually took inspiration and reworded and simplified these sections even further in the last commit, let me know what you think